src/opnsense/www/js/opnsense_ui.js

```diff
@@ -153,6 +153,43 @@ function updateServiceStatusUI(status) {
     $('#service_status_container').html(status_html);
 }
 
+function updateServiceControlUI(ServiceName)
```

Comment:
variables should not start with a uppercase character -> would indicate a class name